test/unit/rules/lint-table-groups-test.js

```diff
@@ -44,6 +44,12 @@ generateRuleTests({
     {{/unless}}
     </table>
     `,
+    '<table>{{some-component tagName="tbody"}}</table>',
```

Comment:
Could you also add some tests that fail?

Reply:
Do you have something specific in mind? I'm not expecting any new failures - this change makes some things that were failing no longer fail.

Reply:
ok this is allowing anything with tagName through. i'll add more cases

Reply:
You could add something like `<table>{{some-component tagName="table"}}</table>`. There should be at least one failure to make sure the new code isn't simply `child.type === 'MustacheStatement'`, without the appropriate `tagName` check. You may also want to add something like `<table>{{some-component someOtherProperty="tbody"}}</table>`

Reply:
updated, let me know if you think I should add any more